packages/amplify-provider-awscloudformation/src/iterative-deployment/deployment-manager.ts

```diff
@@ -199,7 +272,8 @@ export class DeploymentManager {
     try {
       await this.deploymentStateManager?.startRollback();
       await this.deploymentStateManager?.startCurrentStep();
-    } catch {
+    } catch (err) {
+      console.log(err);
```

Comment:
remove `console.log`, perhaps we could log this error with the logger for troubleshooting.